cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -357,13 +346,13 @@ function algoliaLocalizedProduct(product, locale, fieldListOverride, baseModel)
                 }
             }
         }
-        if (algoliaFields.indexOf('id') >= 0) {
+        if (fieldList.indexOf('name') >= 0) {
             this._tags = ['id:' + product.ID];
```

Comment:
Since this line assigns the ID, which we always have, do we still need the condition?

Reply:
This line sets a `_tags` property in the records. I think it's too bad to always add it to the payload when we have a limited amount of fields such as in the specialized jobs.

Reply:
But we could use a different condition if you have a better idea.

Reply:
I tried thinking of a better way to determine within the method whether the model is for a partial update or not, but I don't see any, unless passing this information to it explicitly, so checking the name should be fine, hope it won't lead to any hidden problems. Do you see any other way to determine if it's a partial or full update?

Reply:
Passing the partial/full record update information can make sense yes, I'll give it a try